src/main/java/org/opentripplanner/graph_builder/module/GtfsModule.java

```diff
@@ -113,7 +113,13 @@ public void buildGraph(
         // because the time zone from the first agency is cached
         graph.clearTimeZone();
 
-        CalendarServiceData calendarServiceData = new CalendarServiceData();
+        CalendarServiceData calendarServiceData;
```

Comment:
This code is the same in two places. I think it should be moved inside of `Graph.java` or another utility class.